tests/e2e/deployment-test-cleanup-cloud-function.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#     Copyright 2021 Dynatrace LLC
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+echo -e "$GCP_PROJECT_ID\nY" | ./scripts/uninstall.sh
+gcloud functions delete "$CLOUD_FUNCTION_NAME"
```

Comment:
this delete is needed if you uninstalled before?

Reply:
It deletes sample app (CLOUD_FUNCTION_NAME="e2e_test_sample_app")